src/morello-install/README.md

```diff
@@ -14,7 +14,7 @@ following command would write the image to a USB stick for use with a Morello
 board:
 
 ```
-dd if=cheribsd-memstick-arm64-aarch64c-22.12.img of=/dev/DISK bs=1048576
+dd if=cheribsd-memstick-arm64-aarch64c-23.11.img of=/dev/DISK bs=1048576
```

Comment:
Something I was meaning to change before:
```suggestion
dd if=cheribsd-memstick-arm64-aarch64c-23.11.img of=/dev/DISK bs=1m
```
It's easier to understand and remember if we use more appropriate units.

Reply:
GNU and BSD use different syntax for units; the former is uppercase, not lowercase. So unfortunately it's that way for a good reason.

Reply:
macOS 14.1.1, Ubuntu 22.04.1, FreeBSD 14.0-CURRENT and CheriBSD 23.11 seem to accept `1M`, but Ubuntu doesn't accept `1m` as you said.

Do you know some BSD system that doesn't accept `1M`?

Reply:
Huh, FreeBSD added it in 2004 (https://github.com/freebsd/freebsd-src/commit/1e1d03d7fc9a41c6b926f2b05d6dd9ca8fa6ecee), but the manpage was never changed to mention uppercase being valid. 1M is probably fine then.